src/shogun/features/DotFeatures.cpp

```diff
@@ -358,19 +365,19 @@ void CDotFeatures::benchmark_dense_dot_range(int32_t repeats)
 			dense_dot_range(out, 0, num, alphas, w, d, 23);
 
 #ifdef DEBUG_DOTFEATURES
-    CMath::display_vector(out, 40, "dense_dot_range");
```

Comment:
@vigsterkr any particular reason why we should not try to improve existing code, even if it is just indentation? Am I missing something here?